pkg/kubewarden/components/Policies/PolicyDetail.vue

```diff
@@ -39,6 +42,24 @@ export default {
     },
   },
 
+  async mounted() {
+    if ( this.value?.metadata?.annotations?.[ARTIFACTHUB_PKG_ANNOTATION] ) {
+      try {
+        const artifactHubPackage = await this.value.artifactHubPackageVersion();
+
+        if ( artifactHubPackage && !artifactHubPackage.error ) {
+          this.policyReadme = JSON.parse(JSON.stringify(artifactHubPackage.readme));
```

Comment:
I would double-check for the existance of the property `readme`, otherwise `JSON.parse(JSON.stringify(undefined))` will throw an error and I don't know how `Markdown` or the app will react to that